src/d_iwad.c

```diff
@@ -43,6 +43,7 @@ static const iwad_t iwads[] = {
     {"freedoom1.wad", doom,       retail,       "Freedoom: Phase 1"              },
     {"freedm.wad",    doom2,      commercial,   "FreeDM"                         },
     {"chex.wad",      pack_chex,  retail,       "Chex Quest"                     },
+    {"chex3v.wad",    pack_chex,  retail,       "Chex Quest 3: Vanilla Edition"  },
```

Comment:
I somehow had in mind that the RC versions had different file names, but I just checked and even the RC2 has the same WAD file name.